src/core/rpc.jl

```diff
@@ -0,0 +1,33 @@
+struct RPC
+    flux::Flux
+    future::Future
+
+    function RPC(flux::Flux, topic, payload=nothing; nodeid=API.FLUX_NODEID_ANY, flags=0)
+        if payload === nothing
+            payload = C_NULL
+        else
+            payload = JSON3.write(payload)
+        end 
+        handle = API.flux_rpc(flux, topic, payload, nodeid, flags)
+        new(flux, Future(handle))
+    end
+end
+
+function Base.fetch(rpc::RPC)
```

Comment:
The `Future` struct has a function called `wait` and then this would be called `fetch`.  Are either of those names Julia idioms?  If not, maybe it makes sense to keep the consistent with each other and even with the underlying C APIs?

Reply:
Yeah these are idiomatic names:

https://docs.julialang.org/en/v1/base/parallel/#Base.wait
https://docs.julialang.org/en/v1/base/parallel/#Base.fetch-Tuple{Task}

I have been trying to reuse these names instead of having to prefix everything with `FluxRM`. 

The `wait` on future as an example discard the result of the future https://github.com/flux-framework/FluxRM.jl/blob/bf5f17dd09a0f14dce24943f56eaeec7b991eb20/src/core/future.jl#L21